content/en/blog/2023/lambda-release.md

```diff
@@ -0,0 +1,54 @@
+---
+title: OpenTelemetry Updates our Lambda Support
+linkTitle: OTel Lambda Updates
+date: 2023-05-15
+author: '[Carter Socha](https://github.com/cartersocha) (Lightstep)'
+---
+
+The Functions-as-a-Service (FAAS) SIG is incredibly excited to announce that the
+OpenTelemetry (OTel) community is now releasing our own Lambda layers and has
+[centralized documentation](https://opentelemetry.io/docs/faas/) on how to
+monitor Amazon Web Services (AWS) Lambdas.
+
+### A Slightly New World
+
+If you've been monitoring Lambdas using OTel for a while now you may be slightly
+confused by this announcement. You might say something like OTel has had a
+[repo for Lambda layers](https://github.com/open-telemetry/opentelemetry-lambda)
+and they've been available on AWS for years.
+
+Which is totally correct. Thankfully, we're not remaking the wheel here.
+However, there are some pre-existing problems that impact our users:
+
+- The OTel Lambda layers were only released as part of the
+  [Amazon Distribution of OTel (ADOT)](https://aws-otel.github.io/) and the
```

Comment:
It's AWS Distro for OpenTelemetry (ADOT)

Reply:
updated - thanks